src/utils/args.test.ts

```diff
@@ -0,0 +1,41 @@
+import { parse, validate, Config } from "./args"
+
+describe("The parse function", () => {
+  const args = {
+    args: {
+      template: "template",
+      output: "/path/to/output.ts",
+      stack: "stack"
+    }
+  }
+
+  test("pulls outs template, output and stack args", () => {
+    expect(parse(args)).toMatchObject({
+      cfnPath: "template",
+      outputPath: "/path/to/output.ts",
+      stackName: "stack",
+    })
+  })
+
+  test("pulls outs output dir correctly", () => {
+    expect(parse(args)).toMatchObject({
+      outputDir: "/path/to",
+    })
+  })
+
+  test("pulls outs output file correctly", () => {
+    expect(parse(args)).toMatchObject({
+      outputFile: "output.ts",
+    })
+  })
+})
+
+describe("The validate function", () => {
+  test("does nothing if the file does exist", () => {
+    expect(() => validate({cfnPath: "./README.md"} as Config)).not.toThrow()
```

Comment:
Small one, but worth creating a file specifically for the test? In the (unlikely but possible) scenario the README might move, this test would suddenly fail.

Reply:
As in, create a file in the setup of the test so that we can guarantee there is a file where we expect? Or just add a file to the repo which we know not to move?

Reply:
I've updated this to write a file in the beforeAll and remove in the afterAll. It also checks that the file which shouldn't exist, doesn't exist so you'll get a better message if that's why that test fails. Open to suggestions on the file names/paths if anyone has any better ideas?